packages/crab-netting/src/CrabNetting.sol

```diff
@@ -7,49 +9,126 @@ contract CrabNetting {
     address usdc;
     address crab;
 
-    mapping(address => uint256) usd_balance;
-    mapping(address => uint256) crab_balance;
+    mapping(address => uint256) public usd_balance;
+    mapping(address => uint256) public crab_balance;
     struct Receipt {
-        address depositor;
+        address sender;
         uint256 amount;
     }
-    Receipt[] deposits;
-    Receipt[] crab_deposits;
+    Receipt[] public deposits;
+    uint256 public depositsIndex;
```

Comment:
I see we are deleting the receipt element in netting, won't it make the index always 0?

Reply:
we are deleting an element, that element will be reset to its default value